crates/wasi/wit/deps/sockets/tcp.wit

```diff
@@ -25,24 +25,24 @@ interface tcp {
 	/// If the IP address is zero (`0.0.0.0` in IPv4, `::` in IPv6), it is left to the implementation to decide which
 	/// network interface(s) to bind to.
 	/// If the TCP/UDP port is zero, the socket will be bound to a random free port.
-	///
+	/// 
```

Comment:
Not a problem with this PR, but I guess I just realized there are competing whitespace styles in the wasi wits, so maybe someone needs to invent `witfmt` so we never have to think about them again